deep_qa/layers/tuple_matchers/tuple_match.py

```diff
@@ -0,0 +1,23 @@
+from keras.layers import Layer
+
+class TupleMatch(Layer):
+    '''
+    TupleMatch layers take two tuples (of either the same size or different sizes), which are passed in as a
+     a list of two tensor inputs, and compare them to determine how well they match.  For example, an answer
```

Comment:
Spacing here is a little off, and you have "passed in as a a list".  Can you give a comment about the expected input shapes here?  I'm not sure what situation you might have where the two tuples have different shapes.